sdk/search/Azure.Search.Documents/src/Generated/Models/QueryLanguage.cs

```diff
@@ -161,9 +161,9 @@ public QueryLanguage(string value)
         public static QueryLanguage ArJo { get; } = new QueryLanguage(ArJoValue);
         /// <summary> Query language value for Danish (Denmark). </summary>
         public static QueryLanguage DaDk { get; } = new QueryLanguage(DaDkValue);
-        /// <summary> Query language value for Norwegian (Normway). </summary>
```

Comment:
I'm surprised the swagger linter didn't complain...?

Reply:
I found a company with this name. Internet does not tell me what the word means.